### ISSUE TITLE
did-insert not triggering tracked changes

### ISSUE THREAD

**Issue by patricklx** [emberjs/ember-render-modifiers#20]
e.g. in the example given in the code comments, the `this.count` is not updated in the template...

https://github.com/emberjs/ember-render-modifiers/blob/master/addon/modifiers/did-insert.js#L29

in the callstack i see that `installModifier` is called with a `untrack` context

probably because of this:
https://github.com/emberjs/ember-render-modifiers/pull/16
?

**Comment by btecu** [comment]
I believe `did-insert` is supposed to trigger when the element is inserted into the DOM.

**Comment by btecu** [comment]
Perhaps you want `did-update`?

**Comment by patricklx** [comment]
No, what i mean is that everything inside the did-insert callback is not in a tracking context. So, if a @tracked property is updated in it, other parts tracking it will not know about the change

**Comment by rwjblue** [comment]
Can you describe the actual issue you are running into? `did-insert` is only called once (upon insertion) so I'm a bit confused about what is actually biting you...
